src/ts_node.ts

```diff
@@ -82,12 +83,12 @@ export function tsPath(root: string, orig: string | undefined): string | undefin
     registerTSNode(root)
     const tsconfig = tsconfigs[root]
     if (!tsconfig) return orig
-    const {rootDirs, outDir} = tsconfig.compilerOptions
-    const rootDir = (rootDirs || [])[0]
-    if (!rootDir || !outDir) return orig
+    const {rootDir, rootDirs, outDir} = tsconfig.compilerOptions
+    const rootDirPath = (rootDirs || (rootDir && [rootDir]) || [])[0]
```

Comment:
could we instead make this `const rootDirPath = rootDir || (rootDirs || [])[0]`? I think `rootDir` should be the default if it is defined. We could improve this with `_.castArray()`, but we can't use lodash here unfortunately.

Reply:
Ohh yeah, that is a lot better. Let me update.